tests/acceptance/repo/build-list-routes-test.js

```diff
@@ -127,32 +92,60 @@ test('build history shows, more can be loaded, and a created build gets added an
     assert.ok(page.builds(2).errored, 'expected the third build to have errored');
 
     assert.ok(page.showMoreButton.exists, 'expected the Show More button to exist');
-
-    assert.equal(page.builds(3).name, 'rarely-used', 'expected the old default branch to show');
```

Comment:
I wonder why this has been removed? I made this assertion specifically to test for some weirdness, but perhaps the weirdness is now gone.

![image](https://cloud.githubusercontent.com/assets/43280/23668762/de1d51c4-0316-11e7-9ee7-6d1fb07f3c0f.png)